device_modules/fdo_sys/sys_utils_linux.c

```diff
@@ -347,7 +347,13 @@ size_t get_file_sz(char const *filename)
 	FILE *fp = fopen(filename, "rb");
 
```

Comment:
use fopen_s

Reply:
fopen_s is only supported with the C11 standard. Compiling with the c11 flag produces more errors. Will be fixed in the next release.